frontend/src/metabase/embedding-sdk/theme/embedding-color-palette.ts

```diff
@@ -59,7 +59,7 @@ export const SDK_TO_MAIN_APP_COLORS_MAPPING: Record<
   shadow: ["shadow"],
   positive: ["success"],
   negative: ["danger"],
-  "text-white": ["text-white"],
+  "text-white": ["text-white", "white"],
```

Comment:
If there are places that use `var(--mb-color-white)` in the SDK, we map `"text-white"` (defaulting to `#FFFFFF`) to `--mb-color-white` too - for compatibility purposes.